README.md

```diff
@@ -1,4 +1,48 @@
 ## Shop This City
+  A mobile friendly application build on iOS Platform using React Native. It allows Users to favourite items from assorted well-known brands by their categories which enables the User to build up their own Pinterest Board. A divine feature that allows a User to go back to the items easily when the item has not been purchased. 
+  
+  # SnapShots
+ 
+ 1. Browse Screen 
+![Simulator Screen Shot - iPhone X - 2019-09-13 at 13 43 06](https://user-images.githubusercontent.com/43800526/64897394-1deb8e00-d638-11e9-912a-2db10d0bc12d.png)
+
+
+ 1. Store Screen  
+ 
+![Simulator Screen Shot - iPhone X - 2019-09-13 at 13 43 31](https://user-images.githubusercontent.com/43800526/64897369-06140a00-d638-11e9-8e65-22e07a1d391b.png)
+
+
+ 1. Brand Screen 
+
+![Simulator Screen Shot - iPhone X - 2019-09-13 at 13 43 34](https://user-images.githubusercontent.com/43800526/64897341-f0064980-d637-11e9-88eb-cbdbe8dfa014.png)
+
+
+ 1. New Arrivals 
+![Simulator Screen Shot - iPhone X - 2019-09-13 at 13 43 46](https://user-images.githubusercontent.com/43800526/64897309-d1a04e00-d637-11e9-8e3f-e0faa4190614.png)
+
+
+ 1. Sale 
+
+![Simulator Screen Shot - iPhone X - 2019-09-13 at 13 43 50](https://user-images.githubusercontent.com/43800526/64897252-9ef65580-d637-11e9-8594-9b9c06664449.png)
+
+
+
+
+# Instructions to run the project 
+
+## IOS
+
+1. react-native run-ios
+
```

Comment:
@yenyin262 I think you should include all the steps after `git clone` to run the project.
Like `pod install` and such.
Who doesn't like pods (tide or otherwise) 😂